concourse/pipelines/bosh-cli/bosh-cli.yml

```diff
@@ -0,0 +1,20 @@
+---
+platform: linux
+
+image: docker:///governmentpaas/bosh-cli
```

Comment:
We've created an updated container based on alpine in https://github.com/alphagov/paas-docker-cloudfoundry-tools/pull/23, and also updated the bosh-cli script to specify the shell (now that bash will no longer be available).